net/test/st/run-1x1.sh

```diff
@@ -23,13 +23,25 @@ set -eu
 # set -x
 # export PS4='+ ${FUNCNAME[0]:+${FUNCNAME[0]}():}line ${LINENO}: '
 
+# Get the default net transport
+XPRT=$(m0_default_xprt)
+
 # The same values as in client-server UT
-ADDR_CONSOLE4CLIENTS="$LNET_PREFIX:298"
-ADDR_CONSOLE4SERVERS="$LNET_PREFIX:299"
-ADDR_CMD_CLIENT="$LNET_PREFIX:300"
-ADDR_DATA_CLIENT="$LNET_PREFIX:318"
-ADDR_CMD_SERVER="$LNET_PREFIX:326"
-ADDR_DATA_SERVER="$LNET_PREFIX:334"
+if [ "$XPRT" = "lnet" ]; then
+	ADDR_CONSOLE4CLIENTS="$NID:12345:42:298"
```

Comment:
Suggestion : Could you do something like this, so that it looks a bit cleaner
```
XPRT=$(m0_default_xprt)

# The same values as in client-server UT
if [ "$XPRT" = "lnet" ]; then
ADDR_PREFIX="$NID:12345:42:"
BASE_PORTNUM=298
else
ADDR_PREFIX="$NID@"
BASE_PORTNUM=3000
fi

ADDR_CONSOLE4CLIENTS="$ADDR_PREFIX$BASE_PORTNUM"
ADDR_CONSOLE4SERVERS="$ADDR_PREFIX$((BASE_PORTNUM+1))"
ADDR_CMD_CLIENT="$ADDR_PREFIX$((BASE_PORTNUM+2))"
ADDR_DATA_CLIENT="$ADDR_PREFIX$((BASE_PORTNUM+3))"
ADDR_CMD_SERVER="$ADDR_PREFIX$((BASE_PORTNUM+4))"
ADDR_DATA_SERVER="$ADDR_PREFIX$((BASE_PORTNUM+5))"
```